### ISSUE TITLE
Relicensing Permission: GPLv2+ → 0BSD

### ISSUE THREAD

**Issue by MerryMage** [MerryMage/dynarmic#512]
**tl;dr: Please reply to this issue with a statement saying "Yes, I give my permission to relicense my contributions to dynarmic under 0BSD."**

I am planning on relicensing dynarmic under a permissive license (0BSD). Since dynarmic contains contributions from more people than myself, I am seeking permission to do so from all who have contributed. You can do so by replying to this GitHub issue.

### Why switch from GPL?

Relicensing from GPL will allow dynarmic to be used in more contexts and by more people – GPL essentially forces users of this library to license the resultant combined binary under the GPL.

Since [0BSD](https://opensource.org/licenses/0BSD) is a very liberal license, I do not forsee having to relicense again in the future.

### What if someone doesn't agree?

I hope that people will agree to this, but that is certainly not a certanty nor is everyone guaranteed to be contactable. I will remove code made by people who do not agree or are uncontactable, and expunge their commits from the git history.

### Person List

- [x] @MerryMage
- [x] @Lioncash
- [x] @Tilka
- [x] @bunnei
- [x] @Subv
- [x] @FernandoS27
- [x] @yuriks
- [x] @jroweboy
- [x] @janisozaur
- [x] @BreadFish64
- [x] @neobrain
- [x] @meme
- [x] @lynn
- [ ] @huwpascoe
- [x] @degasus
- [x] @VelocityRa
- [x] @VPeruS
- [x] @Thog
- [ ] @Phanto-m
- [ ] @Annomatg

**Comment by FernandoS27** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by FernandoS27** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by jroweboy** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by ernadeau** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by bunnei** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by lioncash** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by neobrain** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by lynn** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by BreadFish64** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by yuriks** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD!

**Comment by Thog** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by VelocityRa** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by Subv** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD

**Comment by Tilka** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by MerryMage** [comment]
Apologies for the ping, but I was made aware that mentions weren't working properly during the GitHub downtime.

@janisozaur @meme @huwpascoe @VPeruS @Phanto-m @Annomatg I would be grateful if you could drop a comment, thanks.

**Comment by janisozaur** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by meme** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by VPeruS** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.

**Comment by MerryMage** [comment]
Code from individuals who have not responded has been removed from the project's git history.

All other GPL licensed code in the repository has been removed.

**Comment by huwpascoe** [comment]
Yes, I give my permission to relicense my contributions to dynarmic under 0BSD.
